test/typing_checks.py

```diff
@@ -0,0 +1,58 @@
+from typing import Dict
+from pydicti import build_dicti, dicti, odicti, Dicti
+from collections import OrderedDict
+
+
+class V:
+    pass
+
+
+v: V = V()
+d: Dict[str, V] = {}
+o: OrderedDict[str, V] = OrderedDict()
+
+
+# ----------------------------------------
+# dicti
+# ----------------------------------------
+
+_: Dict[str, V] = dicti()
```

Comment:
This is such a nice way to write type tests! I would not have thought of it...

Reply:
Thanks! Not sure it's the recommended way or if it has any downsides though...